pyrate/core/phase_closure/closure_check.py

```diff
@@ -22,18 +22,19 @@
 import pyrate.constants as C
 from pyrate.core import mpiops
 from pyrate.core.phase_closure.mst_closure import sort_loops_based_on_weights_and_date, WeightedLoop, Edge
-from pyrate.configuration import Configuration
+from pyrate.configuration import Configuration, MultiplePaths
 from pyrate.core.phase_closure.sum_closure import sum_phase_closures
 from pyrate.core.phase_closure.plot_closure import plot_closure
 from pyrate.core.shared import Ifg
 from pyrate.core.logger import pyratelogger as log
 
 
-def detect_pix_with_unwrapping_errors(ifgs_breach_count: NDArray[(Any, Any, Any), UInt16],
+def mask_pixels_with_unwrapping_errors(ifgs_breach_count: NDArray[(Any, Any, Any), UInt16],
```

Comment:
This function is not quite doing the expected behaviour.

**Current behaviour**: the function calculates an aggregated number of errors and then masks pixels in all interferograms if the number of errors exceeds the user defined threshold. Every interferogram receives the same masking.

**Expected behaviour:** the function should apply an independent mask layer to each interferogram. The mask should be derived based on the test: `ifgs_breach_count[:, :, i] == num_occurrences_each_ifg[i]` for that interferogram: if True then replace with NaN. Based on this change, the param PHS_UNW_ERR_THR may not be required after all.